enarx-keep-sev-shim/src/usermode.rs

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: Apache-2.0
+
+//! switch to Ring 3 aka usermode
+
+extern "C" {
+    /// Enter Ring 3
+    ///
+    /// # Safety
+    ///
+    /// Because the caller can give any `entry_point` and `stack_pointer`
+    /// including 0, this function is unsafe.
+    pub fn usermode(instruction_pointer: u64, stack_pointer: u64) -> !;
+}
```

Comment:
I don't see `usermode()` actually called anywhere. Can we give it a separate commit?

Reply:
we can .. I just wanted to commit anything GDT related here..

Reply:
No worries.